src/averageAges.js

```diff
@@ -37,7 +53,29 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let count = 0;
+
+  const womenArr = people.filter((person) => {
```

Comment:
Use a more descriptive variable name instead of `womenArr`. Consider using `womenArray` for better readability.